class UserNotFound(Exception):
    pass

class ExerciseNotFound(Exception):
    pass